message-router/message-router.yml

```diff
@@ -1,5 +1,5 @@
-tagline: Route message to different output channels
-desc: A message router consumes a message and republishes it to a different message channel depending on a set of conditions.
-helps: Ballerina enables routing messages to different endpoints by reading the message and using basic control structures such as if and switch statements.
-tags: ["Message Router", "Message Channnel", "Message Endpoint"]
+tagline: Route messages to different output channels
+desc: Message router consumes a message from one channel and republishes it to a different channel depending on a set of conditions.
+helps: "Ballerina supports conditional logic with if-else and match statements. This can be used to route messages based on message content, header, or any custom logic. Ballerina type system supports union types (e.g.: Country type below) which helps to define choices in a type-safe and readable manner." 
```

Comment:
```suggestion
helps: "Ballerina supports conditional logic with if-else and match statements. This can be used to route messages based on message content, header, or any custom logic." 
```

Reply:
We cannot add about union type because in this sample we haven't defined any union type.

Reply:
County is an enum.

Reply:
Enum is one of the types of unions in Ballerina.

Reply:
(`Country` enum is a union)